.gitignore

```diff
@@ -158,3 +158,7 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+notebooks/sg_argosim_notebook.ipynb
+array_baselines.pdf
+clean_observation.pdf
+observation.pdf
```

Comment:
The three pdf files are not included in the git ignore because they are used for building the documentation. If you modified any of this files in your fork, please don't add them to the PR.

Reply:
I have not touched these files, but git automatically added them to the commit, so I wrongfully assumed they were produced by some notebook or similar. I'll remove the changes to .gitignore